.travis.yml

```diff
@@ -30,7 +30,7 @@ script:
   - if test "$TRAVIS_OS_NAME" == "linux"; then export PATH="/home/travis/.dotnet":"$PATH"; fi
   - if test "$TRAVIS_OS_NAME" == "osx"; then export PATH="/Users/travis/.dotnet":"$PATH"; fi
   - dotnet --info
-  - dotnet build -c Release
+  - dotnet build -c Release /p:SourceRevisionId=$TRAVIS_COMMIT /p:GitBranch=$TRAVIS_BRANCH
```

Comment:
Why is that necessary?

Reply:
To pass `props` to `dotnet build` to use current commit and current branch of executing CI that available as CI variables.
 These  `props` are defined in `OrchardCore.Commons.props`  to generate `commit` attribute for nuget package.

`SourceRevisionId` prop is used to have `Product Version` in DLL properties.

Reply:
@sebastienros Using `IncludeSourceRevisionInInformationalVersion`